alembic/versions/e05e1b04de87_add_koji_build_tags.py

```diff
@@ -0,0 +1,61 @@
+"""Add Koji build tags
+
+Revision ID: e05e1b04de87
+Revises: 487a77980f7e
+Create Date: 2024-05-30 14:03:28.962664
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e05e1b04de87"
+down_revision = "487a77980f7e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "koji_build_tags",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("task_id", sa.String(), nullable=True),
+        sa.Column("koji_tag_name", sa.String(), nullable=True),
+        sa.Column("project_id", sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["project_id"],
+            ["git_projects.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_koji_build_tags_task_id"), "koji_build_tags", ["task_id"], unique=False
+    )
+    op.create_index(
+        op.f("ix_koji_build_tags_koji_tag_name"),
+        "koji_build_tags",
+        ["koji_tag_name"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_koji_build_tags_project_id"),
+        "koji_build_tags",
+        ["project_id"],
+        unique=False,
+    )
+    op.execute("ALTER TYPE projecteventtype ADD VALUE 'koji_build_tag'")
```

Comment:
There is no easy way to remove a value from an enum type, so there is no corresponding reverse operation in `downgrade()`. Could that be a problem?

Reply:
I don't think so, I think in the past we did the same for other migrations